src/osd/scheduler/mClockScheduler.cc

```diff
@@ -17,7 +17,13 @@
 #include <functional>
 
 #include "osd/scheduler/mClockScheduler.h"
-#include "common/dout.h"
+#include "common/debug.h"
+
+#if defined(WITH_SEASTAR) && !defined(WITH_ALIEN)
```

Comment:
WITH_ALIEN is deprecated, please rebase.
WITH_SEASTAR will be renamed soon this might break the build when this PR is would go through testing.

Reply:
> WITH_ALIEN is deprecated, please rebase.

Dropped in order to adjust to your API changes (that were merged 5 weeks after I submitted this PR) - it is unfortunate that this PR was ignored for so long, while Ceph moves on, because that causes unnecessary extra work for me.

> WITH_SEASTAR will be renamed soon this might break the build when this PR would go through testing.

Same here. Don't delay this PR, because this causes me extra work. (And I don't have much time left for getting my Ceph improvements merged, as it has been decided that we migrate away from Ceph.)